client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -213,19 +177,74 @@ const DownloadUsersButton = styled(AnchorButton)`
 `
 
 const SupportUserTools = () => {
+  const createOrganization = useCreateOrganization()
+
+  const { register, handleSubmit, reset, formState, getValues } = useForm<{
+    name: string
+  }>()
+
+  const onSubmitCreateOrganization = async ({ name }: { name: string }) => {
+    try {
+      await createOrganization.mutateAsync({ name })
+      reset()
+    } catch (error) {
+      // Do nothing - errors toasted by queryClient
+    }
+  }
+
+  console.log(getValues('name'), formState)
+
   return (
     <Column>
-      <H3>Downloads</H3>
-      <DownloadUsersButton
-        icon="download"
-        href="/api/support/organizations/users"
+      <H3>Tools</H3>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          gap: '32px',
+          paddingTop: '24px',
+        }}
       >
-        Download User List
-      </DownloadUsersButton>
-      <p>
-        Export a list of Audit Admins and Jurisdiction Managers for all audits
-        completed in the last 12 weeks.
-      </p>
+        <form
+          style={{
+            display: 'flex',
+            // flexDirection: 'column',
+            gap: '8px',
+          }}
+          onSubmit={handleSubmit(onSubmitCreateOrganization)}
+        >
+          <Button
+            type="submit"
+            // icon="insert"
+            style={{}}
+            loading={formState.isSubmitting}
+            className={Classes.BUTTON}
```

Comment:
Is the class name needed? I would expect the component to already have its styles baked in